nest/06.glossary.md

```diff
@@ -0,0 +1,182 @@
+
+* `finest group remove <name> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'group' entry of this glossary)
+The command of `finest` to remove a [group](), with `@root` as a parent by default.
+
+* `finest merge`
+
+[//]: # (TODO: add link to the 'Dependency graph' and 'transaction' entries of this glossary)
+The command of `finest` to merge the [dependency graph]() `finest` altered with the [transactions]() you applied, with the dependency graph currently in use by the OS.
+
+* `finest requirement add <requirement> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'requirement' and 'group' entries of this glossary)
+The comand of `finest` to add a [requirement]() to a [group](), by default `@root`.
+
+* `finest requirement remove <requirement> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'requirement' and 'group' entries of this glossary)
+The command of `finest` to remove a [requirement]() from a [group](), by default `@root`.
+
+## G
+[//]: # (TODO: add link to the section 4.1.4)
+* Groups [section 4.1.4]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+Groups are a way to group [packages]() together, in order to install and uninstall them in a more efficient manner.
+## H
+
+## I
+
+## J
+
+## K
+
+## L
+
+## M
+* Mirror
+
+A mirror is a server with a copy of all the data from a main server.
+
+## N
+[//]: # (TODO: add link to the chapter 2)
+* Nest [chapter 2]()
+
+[//]: # (TODO: add link to the 'package manager' entry of this glossary)
+Nest is a dependency-based [package manager]() for Raven-OS.
+It consists of two binaries: `nest` and `finest`.
+
+* `nest`
+
+The binary of Nest mainly targeted towards newcomers.
+
+* `nest install <package>`
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+The command of `nest` to install a [package]().
+
+* `nest list`
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+The command of `nest` to list all installed [packages].
+
+* `nest pull`
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+The command of `nest` to update the list of available [packages] that you can install with Nest.
+
+* `nest uninstall <package>`
+
+The command of `nest` to uninstall a package.
+
+## O
+
+## P
+[//]: # (TODO: add link to the section 1.1)
+* Package [section 1.1]()
+
+A package is a compressed file archive containing all the files required to install, uninstall and update a particular software or application.
+
+[//]: # (TODO: add link to the section 1.1)
+* Package manager [section 1.1]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+A package manager allows the user to install, upgrade or remove software [packages]() on their OS, in other words, to manage packages.
+
+## Q
+
+## R
+[//]: # (TODO: add link to the section 1.3)
+* Repository [section 1.3]()
+
+[//]: # (TODO: add link to the 'package' and 'category' entries of this glossary)
+A repository contains [packages].
```

Comment:
```suggestion
A repository is a collection of [packages] maintained by the same person or organisation, with all [packages]() usually following with the same policy and rules.
```

Reply:
```suggestion
A repository is a collection of [packages] maintained by the same person or organisation, with all [packages]() usually following the same policy and rules.
```
(you put a "with" to much, idk if I should edit your comment or add a new one like this)